tsconfig.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
once you install typescript on your system `npm i -g typescript` you can `tsc --init` to generate these tsconfig linter rules, the only default I changed was `noImplicitAny: false`